Example/NativeBase-KitchenSink/src/screens/toast/toast-type.js

```diff
@@ -41,7 +42,10 @@ class ToastType extends Component {
             onPress={() =>
               Toast.show({
                 text: "Wrong password!",
-                buttonText: "Okay"
+                buttonText: "Okay",
+                onClose: () => {
+                  TLTRN.logScreenLayout("Wrong password!");
```

Comment:
Is there a way to capture automatically? So customer does not need to add one line in onClose?